generic3g/specs/FieldSpec.F90

```diff
@@ -120,15 +126,18 @@ function new_FieldSpec_geom(unusable, geom, vertical_geom, vertical_dim_spec, ty
       type(VerticalDimSpec), intent(in) :: vertical_dim_spec
       type(ESMF_Typekind_Flag), intent(in) :: typekind
       type(UngriddedDims), intent(in) :: ungridded_dims
-
       character(*), optional, intent(in) :: standard_name
       character(*), optional, intent(in) :: units
       character(*), optional, intent(in) :: long_name
       type(StringVector), optional, intent(in) :: attributes
+      type(EsmfRegridderParam), optional, intent(in) :: regrid_param
 
       ! optional args last
       real, optional, intent(in) :: default_value
 
+      type(ESMF_RegridMethod_Flag), allocatable :: regrid_method
+      integer :: status
+
```

Comment:
While we are here anyway, probably best to make the comment about optional args correct by moving it up.

Reply:
`regrid_method` is a local variable here.